packages/runtime-core/src/index.ts

```diff
@@ -22,9 +23,13 @@ export {
   shallowReactive,
   shallowReadonly,
   markRaw,
-  toRaw
+  toRaw,
+  // effect
+  effect,
+  effectScope,
+  isEffect,
+  stop
```

Comment:
Not sure if these APIs can be/should be exported. 

But with the RFC, using `effect` inside `setup()` or other scopes will also be auto collected and disposed. Exporting them could also benefit some libraries that only rely on `@vue/reactivity` to be interchangeable for both Vue and non-Vue environment.